src/player/player.py

```diff
@@ -1,6 +1,7 @@
 import pygame
 
-# from src.map.map import SCROLL_SPEED
+Y = 1
+X = 0
```

Comment:
what are those? x y i guess player's x and y yes?

Reply:
they are indexes. In received tables on index 0 is X and on index 1 is Y